File: turing.py
print("Simple Turing Test Chatbot")
print("Type 'exit' to stop the program")

while True:

    user_input = input("\nAsk something: ").lower()

    if "exit" in user_input:
        print("AI: Goodbye!")
        break

    elif "hello" in user_input or "hi" in user_input:
        print("AI: Hello! Nice to talk with you.")

    elif "name" in user_input:
        print("AI: I am a simple AI chatbot.")

    elif "weather" in user_input:
        print("AI: The weather seems pleasant today.")

    elif "how are you" in user_input:
        print("AI: I am functioning perfectly!")

    else:
        print("AI: Sorry, I don't understand that yet.")

File: turing.c
#include <stdio.h>
#include <string.h>

int main() {

    char input[100];

    printf("Simple Turing Test Chatbot\n");
    printf("Type 'exit' to stop the program\n");

    while(1) {

        printf("\nAsk something: ");
        fgets(input, sizeof(input), stdin);

        if(strstr(input, "exit") != NULL) {
            printf("AI: Goodbye!\n");
            break;
        }

        else if(strstr(input, "hello") != NULL || strstr(input, "hi") != NULL) {
            printf("AI: Hello! Nice to talk with you.\n");
        }

        else if(strstr(input, "name") != NULL) {
            printf("AI: I am a simple AI chatbot.\n");
        }

        else if(strstr(input, "weather") != NULL) {
            printf("AI: The weather seems pleasant today.\n");
        }

        else if(strstr(input, "how are you") != NULL) {
            printf("AI: I am functioning perfectly!\n");
        }

        else {
            printf("AI: Sorry, I don't understand that yet.\n");
        }
    }

    return 0;
}